### ISSUE TITLE
Error while loading server with this mod installed (1.12.2)

### ISSUE THREAD

**Issue by EnhancedC** [ExtraCells/ExtraCells2#670]
I'm not sure if its this mode creating the problem but if any one can help please do
I have reinstalled AE2,Extra cells2 and AE2WTLib and still encountered the problem

[latest.log](https://github.com/ExtraCells/ExtraCells2/files/3746653/latest.log)

[13:17:47] [Server thread/INFO] [extracells]: Enable integration for 'AE2 Wireless Crafting Terminal (wct)'
[13:17:47] [Server thread/ERROR] [net.minecraft.server.MinecraftServer]: Encountered an unexpected exception
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Extra Cells 2 (extracells)

Caused by: java.lang.NoClassDefFoundError: extracells/item/ItemWirelessTerminalUniversal$
	at extracells.proxy.CommonProxy.registerItems(CommonProxy.java:93) ~[CommonProxy.class:?]
	at extracells.ExtraCells.preInit(ExtraCells.java:71) ~[ExtraCells.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_231]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_231]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136) ~[LoadController.class:?]
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:629) ~[Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333) ~[FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:125) ~[nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_231]
Caused by: java.lang.ClassNotFoundException: extracells.item.ItemWirelessTerminalUniversal$
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_231]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_231]
	at extracells.proxy.CommonProxy.registerItems(CommonProxy.java:93) ~[CommonProxy.class:?]
	at extracells.ExtraCells.preInit(ExtraCells.java:71) ~[ExtraCells.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_231]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197) ~[forge-1.12.2-14.23.5.2847-universal.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_231]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_231]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_231]
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116) ~[minecraft_server.1.12.2.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:217) ~[minecraft_server.1.12.2.jar:?]
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136) ~[LoadController.class:?]
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:629) ~[Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333) ~[FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.init(DedicatedServer.java:125) ~[nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486) ~[MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) ~[?:1.8.0_231]
[13:17:47] [Server thread/WARN] [org.dimdev.utils.ModIdentifier]: Failed to identify sun.reflect.GeneratedMethodAccessor9 (untransformed name: sun.reflect.GeneratedMethodAccessor9)
[13:17:47] [Server thread/ERROR] [net.minecraft.server.MinecraftServer]: This crash report has been saved to: C:\Users\dawso\Desktop\LapitosGalacticraftModpack\.\crash-reports\crash-2019-10-19_13.17.47-server.txt
[13:17:47] [Server thread/INFO] [net.minecraft.server.MinecraftServer]: Stopping server
[13:17:47] [Server thread/INFO] [net.minecraft.server.MinecraftServer]: Saving worlds
[13:17:47] [Server thread/INFO] [STDERR]: [java.lang.ThreadGroup:uncaughtException:-1]: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Applied Energistics 2 (appliedenergistics2)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: Caused by: java.lang.NullPointerException
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at appeng.core.AppEng.serverStopped(AppEng.java:271)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.Loader.serverStopped(Loader.java:852)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraftforge.fml.common.FMLCommonHandler.handleServerStopped(FMLCommonHandler.java:508)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:587)
[13:17:47] [Server thread/INFO] [STDERR]: [net.minecraftforge.fml.common.EnhancedRuntimeException:printStackTrace:92]: 	at java.lang.Thread.run(Unknown Source)
[13:17:47] [Server Shutdown Thread/INFO] [net.minecraft.server.MinecraftServer]: Stopping server

**Comment by EnhancedC** [comment]
Mod versions 
appliedenergistics2-rv6-stable-7
ExtraCells-1.12.2-2.6.2a
AE2WTLib-1.12.2-1.0.30

**Comment by ruifung** [comment]
Test again with 2.6.4 and see if its still happening. It shouldn't.

**Comment by ruifung** [comment]
And besides, that error is saying extra cells can't even find one of its *own* classes.

**Comment by EnhancedC** [comment]
> And besides, that error is saying extra cells can't even find one of its _own_ classes.

im still getting the same error 
[latest.log](https://github.com/ExtraCells/ExtraCells2/files/3746761/latest.log)

**Comment by ruifung** [comment]
Could you post the actual crash report mentioned in the log?

**Comment by ruifung** [comment]
I have attempted to reproduce it with a minimum mod install on a server, I am unable to reproduce it.

Perhaps its some class transformer (another coremod) that is breaking it?

**Comment by EnhancedC** [comment]
This the one?
[crash-2019-10-19_14.57.24-server.txt](https://github.com/ExtraCells/ExtraCells2/files/3746906/crash-2019-10-19_14.57.24-server.txt)

**Comment by ruifung** [comment]
I see you're using 2.6.3. Please try with 2.6.4 and let me know if it helps.

EDIT: Even using 2.6.3 version that is currently uploaded to curseforge, I am unable to replicate the issue.

EDIT 2: Can you reproduce it with a *BARE MINIMUM INSTALL* (Nothing but AE2 and extracells only?) If that doesn't crash, start adding mods until you find the one that causes it.

**Comment by EnhancedC** [comment]
> I see you're using 2.6.3. Please try with 2.6.4 and let me know if it helps.
> 
> EDIT: Even using 2.6.3 version that is currently uploaded to curseforge, I am unable to replicate the issue.
> 
> EDIT 2: Can you reproduce it with a _BARE MINIMUM INSTALL_ (Nothing but AE2 and extracells only?) If that doesn't crash, start adding mods until you find the one that causes it.

Ok i found out when i removed 'More commands' mod it successfully launched would you have any idea why, all it did was add more commands

**Comment by ruifung** [comment]
From a quick look at the source of that mod, it does seem to be tampering with the class loading and doing some level of bytecode manipulation. 

When I installed in on my test server it crashed with a class not found.. on "the one probe".
I'll chalk this up as MoreCommands doing some class loading trickery causing stuff to break.

Just FYI, I've seen crash report issues on the issue tracker for that mod referencing spongeforge, immersive engineering, vanillafix... so yeah...

EDIT: I've tried a ABSOLUTE minimum setup with only 3 mods: AE2, EC2 and More Commands. That seems to start fine. Looks like the presence of MoreCommands along with another mod / mods / coremod is mangling the classpath / class bytecode. Nothing I can do on this one.

Why on earth does a mere "command mod" need to do classloading trickery is beyond me.
